index.js

```diff
@@ -12,16 +12,17 @@ module.exports = function (gulp) {
   $.path = require('path');
   $.sassModuleImporter = require('sass-module-importer');
 
-  const tasks = [
+  let tasks = [
     'clean',
+    typeof(pkg.gulpPaths.fractalConfig) == 'undefined' ? false : 'fractal',
```

Comment:
Please always use strict checks when possible. Also should we consider to check, if the file actually exists?

Reply:
Yes probably. I thought about having an `enabledModules` array where we turn Fractal on/off and a file existence check could be done there. Then we send through `enabledModules` to the tasks on [line 27](https://github.com/netzstrategen/gulp-task-collection/pull/7/files/9066363ad6406f8c0e0741e7dc7922071a3a59f7#diff-168726dbe96b3ce427e7fedce31bb0bcR27) so it can be used in the `default` task, for example. That would give us just one place where we're checking/enabling Fractal (or other modules).

Not sure if that's a good solution though?